tests/gold_tests/pluginTest/header_rewrite/gold/cond-elif.gold

```diff
@@ -1,5 +1,5 @@
 ``
-> GET http://www.example.com/from_path/hrw-sets.png``
+> GET http://www.example.com/from_1/hrw-sets.png``
```

Comment:
Unique URL is helpful, but let's not use sequential numbers or letters. Inserting tests would be painful.

Reply:
How would that be painful? I guess I don't really care what we name them, but they have to be named something :). I'd imagined that if you are adding a new test to an existing autest, you append to the current list of remap rules.  Sequential numbering. There'd never be a reason to add a rule in the middle of remap rules I don't think.

We could automate the numbering, but when I looked at that, it makes making the subsequent tests more difficult (you need some sort of identifier).

Reply:
> There'd never be a reason to add a rule in the middle of remap rules I don't think. If there is, at some point, just name it /from_x_y ? E.g. /from_4_1.

I think similar test cases should be close each other. We could do "4_1", but that doesn't completely resolve the issue. Let's say we have "4_1", "4_2", and "4_3". If I wanted to insert a test between "4_1" and "4_2", I'd have to make "4_1_1" or renumber "4_2" and "4_3".

Also even for the case that I append tests, somebody else could be appending another tests on the same file. Then the number would conflicts. Either of those have to update the test number.

I don't think whether tests or remap rules matters in this discussion.